src/pages/POS/SavedInvoiceModal.vue

```diff
@@ -168,6 +187,16 @@ export default defineComponent({
         },
       ] as Field[];
     },
+    filteredInvoices() {
+      const invoices = this.savedInvoiceList
+        ? this.savedInvoices
+        : this.submittedInvoices;
+      return invoices.filter((invoice) =>
+        invoice.name
+          .toLowerCase()
+          .includes(this.invoiceSearchTerm.toLowerCase())
```

Comment:
```suggestion
        : this.submittedInvoices;
      return invoices.filter((invoice) =>
        (invoice.name as string)
          .toLowerCase()
          .includes(this.invoiceSearchTerm.toLowerCase())
```

resolve the type issues